build.js

```diff
@@ -301,9 +303,16 @@ function fullBuild (opts) {
       }
       const filteredLocales = locales.filter(file => junk.not(file) && (selectedLocales ? selectedLocales.includes(file) : true))
       const localesData = generateLocalesData(filteredLocales)
+
+      // This is a list of parallel tasks to be run to save time
+      const asyncTasks = []
       filteredLocales.forEach((locale) => {
-        buildLocale(source, locale, { preserveLocale, localesData })
+        asyncTasks.push(buildLocale(source, locale, { preserveLocale, localesData }))
```

Comment:
Instead of using `forEach` + `push` here, you could use the `map` function which does that for you 🙌 

```js
// This is a list of parallel tasks to be run to save time
const asyncTasks = filteredLocales.map((locale) => (
  buildLocale(source, locale, { preserveLocale, localesData })
))
```

Reply:
Thanks! This is fixed.